utm.yaml

```diff
@@ -983,9 +987,140 @@ components:
         operational intent may not create a conflict with a higher-priority operational intent.  A higher-priority
         operational intent may create a conflict with a lower-priority operational intent.  The regulator specifies
         whether an operational intent may create a conflict with other operational intents of the same priority.
+
+        The priority represents the type of flight and is required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION
+        (EU) 2021/664, paragraph 3. The following values are allowed:
+        - 0: Normal operation.
+        - 100: Special operations as defined in accordance with Article 4 of Implementing Regulation (EU) No 923/2012.
+      enum:
+      - 0
+      - 100
       type: integer
       default: 0
 
+    FlightAuthorisation:
+      description: |-
+        Flight authorisation of an operational intent. It includes mandatory information required by ANNEX IV of COMMISSION
+        IMPLEMENTING REGULATION (EU) 2021/664 for an UAS flight authorisation request.
+
+        Reference: https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:32021R0664&from=EN#d1e32-178-1
+      required:
```

Comment:
I believe in our last meeting we had identified emergency_procedure_url as mandatory.

Reply:
Already fixed by commit 3e3c53c